nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/PutHDFS.java

```diff
@@ -465,10 +465,16 @@ public void process(InputStream in) throws IOException {
                 } catch (final IOException e) {
```

Comment:
Since we throw the GSSException wrapped in a ProcessException, I think we have to catch the ProcessException, then check if it's a GSSException. If so rollback, othwerwise throw it away.